users/views.py

```diff
@@ -47,6 +49,52 @@ class ConfirmationComplete(TemplateView):
     template_name = "registration/confirmation_complete.html"
 
 
+def get_books_borrowed(user):
+    q0 = (
+        BookHolder.objects.select_related("book")
```

Comment:
On the same subject, why did you choose to use `.values()` and not fetch the model (so in your template you would do `loan.book.thumb` instead of `loan.book__thumb`...)?
In this scenario `select_related` would make sense, as it would make the query as efficient.

Reply:
I seemed not to be able to annotate without values() - is that also a case for F? I have to look closer at the different options again I think!

Reply:
Have been looking into this again. I can annotate, but I cannot reduce the bookholder records to just one per title - filter out only the latest record of a given book. That is why I had to use values, I have not figured out another way, although there probably is.